bin/check-ports.rb

```diff
@@ -86,12 +86,19 @@ def run
         port
       end
     end
+
+    hosts = config[:hosts].split(',').flat_map do |host|
```

Comment:
why do we need anything more than `config[:hosts].split(',')`? Also we could auto do this in the argument proc like this:
```ruby
option :hosts,
       short: '-H HOSTNAMES',
       long: '--hostnames HOSTNAMES',
       description: 'Hosts to connect to, comma or semicolon separated list of hosts',
       default: '0.0.0.0',
       proc: proc { |hosts| hosts.split(//[,;]\s*//) },
```

Reply:
Fixed this